js/app.js

```diff
@@ -21,3 +25,68 @@ function addEvent(eventTagId) {
 eventTagId.forEach(
   addEvent,
 );
+
+const projectContent = [{
+  title: 'Tonic',
+  text: "Tonic is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and .",
+  mobileImage: './img/card1.png',
+  tech: ['html', 'css', 'javaScript'],
+},
+{
+  title: 'Multi-Post Stories',
+  text: "Multi-Post Stories is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and ",
+  mobileImage: './img/card2.svg',
+  tech: ['html', 'css', 'javaScript', 'Angular'],
```

Comment:
- [ ] As it is a requirement for this project, please make sure each object contains at least the following information about the project: `name`, `description`, `featuring image`, `technologies`, `link to live version`, and link to the `project source code`. Please the last two fields are missing, kindly add them as this make adding projects in the future easy, you will only add it in one place - as another object in the projects array.